html/pfappserver/lib/pfappserver/I18N/en.po

```diff
@@ -2716,6 +2731,10 @@ msgstr ""
 msgid "Interval at which PacketFence updates the online node count"
 msgstr ""
 
+# conf/documentation.conf (maintenance.option82_query_interval)
+msgid "Interval at which PacketFence will query compatibles switches for option 82 switch mac address"
```

Comment:
Interval at which PacketFence will query compatible switches for DHCP Option 82 switch MAC address